### ISSUE TITLE
mocked rest operations with a script dispatcher are broken after an update

### ISSUE THREAD

**Issue by ymeudal** [microcks/microcks#877]
### Describe the bug

I have a rest operation, defined in postman by a GET {{baseurl}}/endpoint/p1/:param1

When I first upload my postman collection, I will have a default URI_PARTS dispatcher, with "param1" as the dispatching rule.
All is good with this default setup, I am able to perform different GETs with my samples.
Then, I would like to change dispatcher type to a SCRIPT, because I have to enhance some replies with some dynamic stuff, or get some extra query parameters.
At this step, all is fine, script is doing what it is expected to do. 
When I upload a second time the same postman collection, all requests return a status code 404 (not found). 

### Expected behavior

I should be able to upload a second time a postman collection to update the existing mock (for adding example, or create new operations...). Current dispatcher should be preserved. Requests to this mock should respond like the first time.

### Actual behavior

I want to upload a second time my postman collection, for any reason. 
Actual script of the dispatcher remains, but it is now impossible to execute same requests: Error 404.

### Microcks version or git rev

1.7.1

### Install method (`docker-compose`, `helm chart`, `operator`, `docker-desktop extension`,...)

docker-compose

**Comment by lbroudoux** [comment]
Hey there! Pushed a fix that should be available in the `quay.io/microcks/microcks:nightly` container image. Please let me know how things are going.

**Comment by ymeudal** [comment]
I am sorry, but I have difficulties to run nightly built in a docker environment. 
With docker-compose.yml from 1.8 branch, I had the error: 
com.mongodb.MongoIncompatibleDriverException: Server at mongo:27017 reports wire version 5, but this version of the driver requires at least 6 (MongoDB 3.6)
I changed mongo: image: mongo:3.6.23, microcks started successfully.

I confirm that the bug has been fixed. SCRIPT dispatcher is still functional after an other collection import. If I want to rollback the dispatcher to a type URI_PARTS, I have to import the collection again, but it is finally OK.

Thanks Laurent!

**Comment by lbroudoux** [comment]
Yes... I found the MongoDB stuff is due to upgrading to Spring Boot 3.1 (see #840). I still need to push those changes. 

Thanks a lot for the feedback regarding the fix! I'm closing this issue and gonna push the Mongo stuffs.
